### PR TITLE
Add power tribe monthly forecast report to prox

### PR DESCRIPTION
**Pull request by mixman**

### CODE REVIEW

File: proxy-types/src/Futurice/App/Proxy/API.hs [hunk 1]
```diff
@@ -132,6 +132,10 @@ data Routes = Routes
         ("uiapi" :> "tribe/" :> Get '[JSON] Value)
         ("power" :> "api" :> "v3" :> "tribes" :> Get '[JSON] Value)
 
+    , routePowerV3TribesMonthlyForecast :: ProxiedEndpoint 'PowerService
+        ("uiapi" :> "tribe" :> "monthly_forecast/" :> Get '[JSON] Value)
+        ("power" :> "api" :> "v3" :> "tribes" :> "monthly" :> "forecast" :> Get '[JSON] Value)
```
Comment (neurogoo) [inline proxy-types/src/Futurice/App/Proxy/API.hs]: For us, it is a little easier if the endpoints are similar than in source. Or is there reason to change that?

Comment (mixman) [reply proxy-types/src/Futurice/App/Proxy/API.hs]: What change are you looking for?

Comment (neurogoo) [reply proxy-types/src/Futurice/App/Proxy/API.hs]: both endpoints to be either "montly_forecast" or "monthly/forecast"

Comment (mixman) [reply proxy-types/src/Futurice/App/Proxy/API.hs]: Ah! Too late for Power without refactoring there.

Comment (mixman) [reply proxy-types/src/Futurice/App/Proxy/API.hs]: Settled offline. Will migrate in future.

**Pull request merged**